examples/release_highlights/plot_release_highlights_1_5_0.py

```diff
@@ -24,89 +24,136 @@
 # %%
 # FixedThresholdClassifier: Setting the decision threshold of a binary classifier
 # -------------------------------------------------------------------------------
-# All binary classifiers of scikit-learn use a fixed decision threshold of 0.5 to
-# convert probability estimates (i.e. output of `predict_proba`) into class
-# predictions. However, 0.5 is almost never the desired threshold for a given problem.
-# :class:`~model_selection.FixedThresholdClassifier` allows to wrap any binary
-# classifier and set a custom decision threshold.
+# All binary classifiers of scikit-learn use a fixed decision threshold of 0.5
+# to convert probability estimates (i.e. output of `predict_proba`) into class
+# predictions. However, 0.5 is almost never the desired threshold for a given
+# problem. :class:`~model_selection.FixedThresholdClassifier` allows wrapping any
+# binary classifier and setting a custom decision threshold.
 from sklearn.datasets import make_classification
+from sklearn.model_selection import train_test_split
 from sklearn.linear_model import LogisticRegression
-from sklearn.metrics import confusion_matrix
+from sklearn.metrics import ConfusionMatrixDisplay
+
 
-X, y = make_classification(n_samples=1_000, weights=[0.9, 0.1], random_state=0)
-classifier = LogisticRegression(random_state=0).fit(X, y)
+X, y = make_classification(n_samples=10_000, weights=[0.9, 0.1], random_state=0)
+X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=0)
 
-print("confusion matrix:\n", confusion_matrix(y, classifier.predict(X)))
+classifier_05 = LogisticRegression(C=1e6, random_state=0).fit(X_train, y_train)
+_ = ConfusionMatrixDisplay.from_estimator(classifier_05, X_test, y_test).plot()
 
 # %%
 # Lowering the threshold, i.e. allowing more samples to be classified as the positive
 # class, increases the number of true positives at the cost of more false positives
 # (as is well known from the concavity of the ROC curve).
 from sklearn.model_selection import FixedThresholdClassifier
 
-wrapped_classifier = FixedThresholdClassifier(classifier, threshold=0.1).fit(X, y)
-
-print("confusion matrix:\n", confusion_matrix(y, wrapped_classifier.predict(X)))
+classifier_01 = FixedThresholdClassifier(classifier_05, threshold=0.1)
+classifier_01.fit(X_train, y_train)
+_ = ConfusionMatrixDisplay.from_estimator(classifier_01, X_test, y_test).plot()
```

Comment:
The plot is duplicated, see https://output.circle-artifacts.com/output/job/b4ff1d34-b905-4fb1-92e1-162fab57e5b6/artifacts/0/doc/auto_examples/release_highlights/plot_release_highlights_1_5_0.html
```suggestion
_ = ConfusionMatrixDisplay.from_estimator(classifier_01, X_test, y_test)
```